test/Microsoft.AspNet.SignalR.FunctionalTests/Client/LongPollingFacts.cs

```diff
@@ -103,15 +105,20 @@ public void LongPollingDoesNotPollAfterTransportIsBeingStoppedMidRequest()
 
             var longPollingTransport = new LongPollingTransport(mockHttpClient.Object);
 
-            Assert.True(
-                longPollingTransport.Start(mockConnection.Object, string.Empty, disconnectCts.Token)
-                    .Wait(TimeSpan.FromSeconds(15)));
+            try
+            {
+                await longPollingTransport.Start(mockConnection.Object, string.Empty, disconnectCts.Token)
+                    .OrTimeout(TimeSpan.FromSeconds(15));
+            }
+            catch (StartException stex) when (stex.InnerException is OperationCanceledException)
```

Comment:
Paging @davidfowl . It seems like this test sometimes throws up an OperationCanceledException *because* of the `disconnectCts.Cancel()` line above, in the test. It only throws that sometimes though, and it seems to be expected that the Start would fail since we're stopping it mid-request.